.azure-pipelines/templates/linuxBuild.yml

```diff
@@ -37,6 +39,10 @@ steps:
       # Building
       mkdir build
       cd build
-      cmake ../ -G Ninja ${{ parameters.extraflags }} -DLOCAL_STATIC_HDF5:bool=true -DHDF5_DIR:path=${HDF5_DIR}
+      cmake ../ -G Ninja ${{ parameters.extraflags }} -DLOCAL_STATIC_HDF5:bool=true -DHDF5_DIR:path=${HDF5_DIR} -Wl,-rpath,/tmp/qt/6.1.2/gcc_64/lib
```

Comment:
Obviously this link command sticks out a bit!  However, happy for it to be left there for now with a view to sorting out "proper" Linux packaging at some future point.